explorer/client/src/utils/utility_functions.tsx

```diff
@@ -0,0 +1,59 @@
+import mockTransactionData from './mock_data.json';
+import { DefaultRpcClient as rpc } from './rpc';
+
+
+const navigateWithUnknown = async (input: string, navigate: Function) => {
+    console.log('searching for ' + input);
```

Comment:
nit: remove this and several other places in the file as well

Reply:
removed: https://github.com/MystenLabs/sui/commit/3ec57a29a5863f26f997e70d658ef87d2c8f5a0f